README.md

```diff
@@ -8,6 +8,21 @@ It includes options to automatically batch all the `.heic` files it finds in a f
 
 [Download the latest release](https://github.com/danbrakeley/heic2png/releases), and then put the exe somewhere in your path.
 
+### Compiling on Linux
+
+1. Install golang
+  - Via apt `apt install golang-go`
+  - Via source [Download and install](https://go.dev/doc/install)
+2. Clone this repository `git clone https://github.com/danbrakeley/heic2png.git`
+3. Enter the repository and execute the [build.sh](https://github.com/danbrakeley/heic2png/blob/main/build.sh) script
+  - ```
+    cd heic2png
+    chmod +x build.sh
```

Comment:
This line shouldn't be necessary. Instead of adding this line, you should set the execute bit as part of this PR. I think you can achieve that via [`git add --chmod=+x build.sh`](https://stackoverflow.com/questions/40978921/how-to-add-chmod-permissions-to-file-in-git)